### PR TITLE
moved respond methods to interface for the message client

### PR DESCRIPTION
**Pull request by lasrinnil**
## Description

The failed and succeeded methods were not moved to the interface. This PR will fix this.

## References

* #588

### CODE REVIEW

**Review by HenrikSommer — APPROVED** [review]

File: source/Energinet.Charges.Libraries/source/Energinet.DataHub.Charges.Clients/DefaultChargeLinkMessages/IDefaultChargeLinkMessagesRequestClient.cs [hunk 1]
```diff
@@ -29,5 +29,27 @@ public interface IDefaultChargeLinkMessagesRequestClient
         Task CreateDefaultChargeLinkMessagesRequestAsync(
```
Comment (prtandrup) [inline source/Energinet.Charges.Libraries/source/Energinet.DataHub.Charges.Clients/DefaultChargeLinkMessages/IDefaultChargeLinkMessagesRequestClient.cs:29]: Please change the end of the line to state "Create default charge link messages"

File: source/Energinet.Charges.Libraries/source/Energinet.DataHub.Charges.Clients/DefaultChargeLinkMessages/IDefaultChargeLinkMessagesRequestClient.cs [hunk 2]
```diff
@@ -29,5 +29,27 @@ public interface IDefaultChargeLinkMessagesRequestClient
...
             [NotNull] CreateDefaultChargeLinkMessagesDto createDefaultChargeLinkMessagesDto,
             [NotNull] string correlationId);
+
+        /// <summary>
+        /// Used by the Charges domain to respond with failed to the request to create default charge links.
```
Comment (prtandrup) [inline source/Energinet.Charges.Libraries/source/Energinet.DataHub.Charges.Clients/DefaultChargeLinkMessages/IDefaultChargeLinkMessagesRequestClient.cs:34]:
```suggestion
        /// Used by the Charges domain to respond with failed to the request to create default charge link messages.
```

File: source/Energinet.Charges.Libraries/source/Energinet.DataHub.Charges.Clients/DefaultChargeLinkMessages/IDefaultChargeLinkMessagesRequestClient.cs [hunk 3]
```diff
@@ -29,5 +29,27 @@ public interface IDefaultChargeLinkMessagesRequestClient
...
+        /// <param name="createDefaultChargeLinkMessagesFailedDto"></param>
+        /// <param name="correlationId"></param>
+        /// <param name="replyQueueName"></param>
+        public Task CreateDefaultChargeLinkMessagesFailedRequestAsync(
+            [NotNull] CreateDefaultChargeLinkMessagesFailedDto createDefaultChargeLinkMessagesFailedDto,
+            [NotNull] string correlationId,
+            [NotNull] string replyQueueName);
+
+        /// <summary>
+        ///  Used by the Charges domain to respond with success to the request to create default charge links.
```
Comment (prtandrup) [inline source/Energinet.Charges.Libraries/source/Energinet.DataHub.Charges.Clients/DefaultChargeLinkMessages/IDefaultChargeLinkMessagesRequestClient.cs:45]:
```suggestion
        ///  Used by the Charges domain to respond with success to the request to create default charge link messages.
```

**Review by prtandrup — APPROVED** [review]
Approved with minor comments

**Pull request merged**